Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.py

```diff
@@ -1106,6 +1108,150 @@ def context_command(client: Client, args: dict, reliability: str) -> CommandResu
     )
 
 
+@exception_handler
+@logger
+def cve_command(client: Client, args: dict, reliability: str) -> CommandResults:
+    """
+    Returns information about CVE.
+
+    :type client: ``Client``
+    :param client: client object
+
+    :type args: ``dict``
+    :param args: All command arguments, usually passed from ``demisto.args()``.
+    :return: A ``CommandResults`` object that is then passed to ``return_results``,
+           that contains the IP information.
+    :rtype: ``CommandResults``
+
+    :type reliability: ``String``
```

Comment:
move it up under 'dict' so all the arguments will be together and then the return value.
you can add an explanation what the reliability is (I have no idea how the string should look like)